app/components/badge-appveyor.js

```diff
@@ -6,8 +6,22 @@ export default Component.extend({
     tagName: 'span',
     classNames: ['badge'],
 
+    id: alias('badge.attributes.id'),
     repository: alias('badge.attributes.repository'),
 
+    imageUrl: computed('badge.attributes.id', function() {
+        let id = this.get('badge.attributes.id');
+        let branch = this.get('branch');
+        if (id === undefined || id === null) {
```

Comment:
I think this condition is backwards? This says if id IS undefined or id IS null, then try to use id? This should be `(id !== undefined && id !== null)`, yeah? I tried making a badge using only `repository` and `id` and the `id` isn't appearing in the image url :(